### DISCUSSION TITLE
One Twilio Visual Language adoption

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#2970] [Office hours] [answered]
Hey Paste team,

I work on the MSGUI team (who owns most of the messaging console experience) and our team is super excited about this new visual language. I do have couple of questions around the adoption of this after reading through the doc you have shared:

1. In the table it says PHASE III:  Update to the new Twilio theme ’s timeline is Q2 - Q4 23. I am wondering if the Paste team will send another message once the code is ready for the team to use?
2. Again, for the PHASE II: Implement Paste's new unified navigation , consider the timeline for this is Q3, could our team implement other stuff first than circle back to this nav once it’s ready?
3. From my understanding, eng wise if we use Paste component, engs just need to keep your Paste dependencies up to date at regular intervals  but what if we have some customized designs, I believe we should do an audit and spend some time to re-create those component in the FE using the new visuals?
4. in short, my team are looking into the possibility of:

- Adopt the new visual language in Q2 (or once the code is ready)
- Adopt the new nav in Q3 (or once the code is ready)
- For non-standard Paste component, how much work / resources we need to spend to make them the new visuals

I have signed up for a paste office hour next week and super excited to work with y'all of getting this new visuals to our messaging customers!

**Comment by loreina** [comment]
Hi @frankfzhao, thanks for the write-up. Our team will follow up here after office hours. Cheers!

**Comment by serifluous** [comment, marked as answer]
Thanks for coming to office hours! Here's the recording: https://www.loom.com/share/88d4a8a39c654454bf0587f3eaeae195 You can find the password in our slack channel.
